tsfeatures/tsfeatures.py

```diff
@@ -117,12 +132,12 @@ def arch_stat(x: np.array, freq: int = 1,
         'arch_lm': R^2 value of an autoregressive model of order lags applied to x**2.
     """
     if len(x) <= lags + 1:
-        return {'arch_lm': np.nan}
+        return {"arch_lm": np.nan}
     if demean:
-        x = x - np.mean(x)
+        x -= np.mean(x)
```

Comment:
This would modify the input, which is an unwanted side-effect.

Reply:
reverted back